src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/ResendStaffNotificationsCoreCaseDataTest.java

```diff
@@ -0,0 +1,248 @@
+import uk.gov.hmcts.cmc.email.EmailData;
+import uk.gov.hmcts.reform.sendletter.api.SendLetterApi;
+import uk.gov.hmcts.reform.sendletter.api.SendLetterResponse;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.cmc.claimstore.repositories.CCDCaseApi.CASE_TYPE_ID;
+import static uk.gov.hmcts.cmc.claimstore.repositories.CCDCaseApi.JURISDICTION_ID;
+import static uk.gov.hmcts.cmc.claimstore.utils.ResourceLoader.listOfCaseDetails;
+import static uk.gov.hmcts.cmc.claimstore.utils.ResourceLoader.listOfCaseDetailsWithDefResponse;
+import static uk.gov.hmcts.cmc.claimstore.utils.ResourceLoader.listOfCaseDetailsWithDefendant;
+
+@TestPropertySource(
+    properties = {
+        "document_management.api_gateway.url=false",
+        "core_case_data.api.url=http://core-case-data-api"
+    }
+)
+public class ResendStaffNotificationsCoreCaseDataTest extends BaseGetTest {
+
+    private static final String CASE_REFERENCE = "000MC023";
+
+    @MockBean
+    protected SendLetterApi sendLetterApi;
+
+    @Captor
+    private ArgumentCaptor<EmailData> emailDataArgument;
+
+    private static final String PAGE = "1";
+
+    @Before
+    public void setUp() {
+        given(pdfServiceClient.generateFromHtml(any(byte[].class), anyMap()))
+            .willReturn(new byte[]{1, 2, 3, 4});
+        UserDetails userDetails = SampleUserDetails.getDefault();
+        User user = new User(AUTHORISATION_TOKEN, userDetails);
+        given(userService.getUserDetails(AUTHORISATION_TOKEN)).willReturn(userDetails);
+        given(userService.authenticateAnonymousCaseWorker()).willReturn(user);
+        given(userService.getUser(AUTHORISATION_TOKEN)).willReturn(user);
+        given(authTokenGenerator.generate()).willReturn(SERVICE_TOKEN);
+    }
+
+    @Test
+    public void shouldRespond404WhenClaimDoesNotExist() throws Exception {
+        final String nonExistingClaimReference = "something";
+        final String event = "claim-issue";
+        given(coreCaseDataApi.searchForCitizen(
+            any(),
+            any(),
+            any(),
+            any(),
+            any(),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE, "sortDirection", "desc"))
+            )
+        ).willReturn(listOfCaseDetails());
+        makeRequest(nonExistingClaimReference, event).andExpect(status().isNotFound());
```

Comment:
Please add few blank lines to let this code breath a bit like you have done in `shouldRespond409AndNotProceedForClaimIssuedEventWhenClaimIsLinkedToDefendant`. Please apply the same pattern to other methods in this test class.